api/docs/CMake_doxyfile.cmake

```diff
@@ -128,6 +128,11 @@ string(REGEX REPLACE
   "(footer.html)"
   "\"${dest_dir}/\\1\"" string "${string}")
 
+# If we end up with more of these we should generalize and pass them in
+# rather than hardcoding here.
+string(APPEND string
+  "HTML_EXTRA_FILES=${proj_bindir}/clients/drcachesim/docs/new-features-encodings-seek.pdf")
```

Comment:
Just curious why creating a presentation pdf was preferred over maybe a detailed entry in release.dox.

Reply:
We had created these slides for presentations already and wanted to place them somewhere they could be referred to later.